src/test/java/core/basesyntax/service/operation/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.operation;
+
+import static core.basesyntax.model.FruitTransaction.Operation.PURCHASE;
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+
+    private static PurchaseOperationHandler purchaseHandler;
+    private FruitTransaction fruitTransaction;
```

Comment:
```suggestion
    private static OperationHandler purchaseHandler;
```
fruitTransaction should be local variable